pkgs/build-support/rust/hooks/cargo-setup-hook.sh

```diff
@@ -61,14 +61,15 @@ cargoSetupPostPatchHook() {
       fi
 
       echo
-      echo "ERROR: cargoSha256 is out of date"
+      echo "ERROR: cargoHash or cargoSha256 is out of date"
       echo
       echo "Cargo.lock is not the same in $cargoDepsCopy"
       echo
       echo "To fix the issue:"
-      echo '1. Use "0000000000000000000000000000000000000000000000000000" as the cargoSha256 value'
-      echo "2. Build the derivation and wait for it to fail with a hash mismatch"
-      echo "3. Copy the 'got: sha256:' value back into the cargoSha256 field"
+      echo '1. Use in your derivation: cargoHash = "";';
+      echo '2. Build the derivation and wait for it to fail with a hash mismatch'
+      echo '3. Copy the "got: sha256-XXXXXXXXX=" value back into the cargoHash field'
+      echo '   This results in: cargoHash = "sha256-XXXXXXXXX=";'
```

Comment:
Maybe the example would do more harm (because it's shorter) than good? Not sure.

```suggestion
      echo '3. Copy the "got" value into the cargoHash field'
```

Reply:
```suggestion
      echo '3. Copy the "got: sha256-..." value back into the cargoHash field'
      echo '   This results in: cargoHash = "sha256-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX=";'
```